app_dart/lib/src/foundation/utils.dart

```diff
@@ -136,6 +138,12 @@ Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> fil
       }
     }
   }
+
+  log.info('Collected the following targets to run:');
+  for (var element in targetsToRun) {
```

Comment:
If this is a part of `targetsToRun`, it would be more readable to call it `target` instead of `element`